e2e/wdio/core/tests/dynamic-page.e2e-spec.ts

```diff
@@ -0,0 +1,335 @@
+            checkLinksClickable(tabsExample);
+        });
+
+        it('should check navigate by link', () => {
+            checkUrlNavigation(tabsExample);
+        });
+
+    });
+
+    describe('Tests for default example', () => {
+
+        it('should check close page by clicking exit button', () => {
+            openPage(defaultExample);
+            checkPageClosed(exitButton);
+        });
+
+        it('should check close page by clicking cancel button', () => {
+            openPage(defaultExample);
+            checkPageClosed(cancelButton);
+        });
+
+        it('should check close page by clicking reject button', () => {
+            openPage(defaultExample);
+            checkPageClosed(rejectButton);
+        });
+
+        it('should check close page by clicking accept button', () => {
+            openPage(defaultExample);
+            checkPageClosed(acceptButton);
+        });
+
+        it('should check close page by clicking save button', () => {
+            openPage(defaultExample);
+            checkPageClosed(saveButton);
+        });
+
+        it('should check close page by clicking close button', () => {
+            openPage(defaultExample);
+            checkPageClosed(closeButton);
+        });
+
+        it('should check links clickable', () => {
+            checkLinksClickable(defaultExample);
+        });
+
+        it('should check navigate by link', () => {
+            checkUrlNavigation(defaultExample);
+        });
+
+    });
+
+    it('should check RTL and LTR orientation', () => {
+        dynamicPagePage.checkRtlSwitch();
+    });
+
+    function checkPageClosed(button: string): void {
+        click(button);
+        expect(doesItExist(dynamicPageContent)).toBe(false, 'dynamic page is not closed');
+    }
+
+    function openPage(section: string): void {
+        click(section + button);
+        expect(isElementDisplayed(dynamicPageContent)).toBe(true, 'dynamic page is not opened')
+    }
+
+    function checkPiningArea(section: string): void {
+        click(section + button);
+        click(pinButton);
+        expect(getAttributeByName(pinButton, 'aria-selected')).toBe('true', 'area is not pinned');
+        const articlesLength = getElementArrayLength(article);
+        scrollIntoView(article, articlesLength - 1);
+        expect(isElementDisplayed(collapsibleHeader)).toBe(true, 'collapsible area is not visible, pinning works incorrect');
+        click(exitButton);
+    }
+
+    function checkCollapsingArea(section: string): void {
+        click(section + button);
+        click(collapseButton);
+        expect(isElementDisplayed(collapsibleHeader)).toBe(false, 'collapsible area is not hidden');
+        click(collapseButton);
+        expect(isElementDisplayed(collapsibleHeader)).toBe(true, 'collapsible area is not displayed');
+        click(exitButton);
+    }
+
+    function checkLinksClickable(section: string): void {
+        click(section + button);
```

Comment:
seems like you don't need this, can just call openPage() before calling checkLinksClickable()
also we already have a method for this in the assertion helper - checkElArrIsClickable(element)

so you can just do this in your test case:
openPage(section);
checkElArrIsClickable(links);